src/ingestion/zengrc.py

```diff
@@ -0,0 +1,67 @@
+import requests
+import base64
+import datetime
+import json
+import os
+
+from runners.helpers import db, log
+
+ZENGRC_ID = os.environ.get('ZENGRC_ID')
+ZENGRC_SECRET = os.environ.get('ZENGRC_SECRET')
+ZENGRC_URL = os.environ.get('ZENGRC_URL')
+ZENGRC_TABLE = os.environ.get('ZENGRC_TABLE')
+TIMESTAMP = str(datetime.datetime.utcnow())
+
+
+def gen_headers(id, secret):
+    auth = base64.b64encode(f'{id}:{secret}'.encode()).decode()
+    return {"Authorization": f"Basic {auth}"}
+
+
+def loop(endpoint):
+    log.info(f"starting {endpoint}")
+    j = {'links': {'next': {'href': endpoint}}}
```

Comment:
what is `j`? is there a better name for it?

Reply:
resolved